### ISSUE TITLE
Analisys Options - Concentration Units

### ISSUE THREAD

**Issue by Ostoveni** [neslerel/BetaTesters_QGRed#45]
Pienso que se debe cambiar la caja de texto por un objeto ComboBox.
Cuando se importe desde QGISRed una red de Epanet, mostrar algún mensaje de aviso sino es una de las dos unidades permitidas.

![MassUnits_Quality](https://user-images.githubusercontent.com/20848965/70187130-c700cc80-16ed-11ea-914d-a0e1f2940a65.png)

**Comment by neslerel** [comment]
Sí, eso hay que hacerlo. ¿Podrías verificar que esas dos opciones son las mismas aunque cambies las unidades de caudal?

**Comment by Ostoveni** [comment]
Según el manual de Epanet, se puede elegir una u otra unidad de medida, sin importar la unidad de caudal elegida

![Mass_Units_Epanet](https://user-images.githubusercontent.com/20848965/70458969-84efd600-1ab3-11ea-8e3e-f5d25e625450.png)
